src/olympia/activity/utils.py

```diff
@@ -136,8 +138,21 @@ def log_and_notify(action, comments, note_creator, version):
         alog.user for alog in ActivityLog.objects.for_version(version)]
     # Collect add-on authors (excl. the person who sent the email.)
     addon_authors = set(version.addon.authors.all()) - {note_creator}
-    # Collect reviewers on the thread (again, excl. the email sender)
-    reviewer_recipients = set(log_users) - addon_authors - {note_creator}
+    # Collect staff that want a copy of the email
+    try:
+        cc_group = models.Group.objects.get(name='Activity Mail CC')
+        staff_cc = [user for user in cc_group.users.all()]
```

Comment:
You're not using `cc_group` later, maybe something like `UserProfile.objects.filter(groups__name='Activity Mail CC')` is more performant than fetching the group first and the users later (two queries, two round trips). Though "performant" is relative here, it's a string match on `name` but we don't have too many groups imho so that should be fine.


Also, we should start (or do we already do that somewhere?) using constants for group names like these and put them into `olympia.constants.{users,group}` or something like that.